### PR TITLE
Add Trino 423 release notes

### LINKED ISSUE #18288: Release notes for 423

### PR DESCRIPTION
**Pull request by colebow**
## Description

Assemble the release notes for the upcoming Trino 423 release. Fixes #18288 

## Release notes

(x) This is not user-visible or docs only and no release notes are required.

## Verification for each pull request

Format: maintainer, PR/issue number, ✅ / ❌ rn ✅ / ❌ docs
✅ rn - release note added and verified, or assessed to be not necessary, set to ❌ rn before completion
✅ docs - need for docs assessed and merged, or assessed to be not necessary, set to ❌ docs before completion

## 13 Jul 2023

* #18157 ✅ rn ✅ docs
* #18276 ✅ rn ✅ docs

## 14 Jul 2023

* #18261 ✅ rn ✅ docs
* #18003 ✅ rn ✅ docs
* #18263 ✅ rn ✅ docs
* #18212 ✅ rn ✅ docs
* #18278 ✅ rn ✅ docs
* #18257 ✅ rn ✅ docs
* #18246 ✅ rn ✅ docs
* #18165 ✅ rn ✅ docs
* #18252 ✅ rn ✅ docs
* #18253 ✅ rn ✅ docs
* #18251 ✅ rn ✅ docs
* #18206 ✅ rn ✅ docs
* #18205 ✅ rn ✅ docs
* #18195 ✅ rn ✅ docs
* #18292 ✅ rn ✅ docs
* #18179 ✅ rn ✅ docs
* #18235 ✅ rn ✅ docs
* #18297 ✅ rn ✅ docs

## 15 Jul 2023

* #18232 ✅ rn ✅ docs

## 17 Jul 2023

* #18306 ✅ rn ✅ docs
* #18300 ✅ rn ✅ docs
* #18301 ✅ rn ✅ docs
* #17853 ✅ rn ✅ docs
* #15606 ✅ rn ✅ docs

## 18 Jul 2023

* #18314 ✅ rn ✅ docs
* #18311 ✅ rn ✅ docs
* #18296 ✅ rn ✅ docs
* #18294 ✅ rn ✅ docs
* #18316 ✅ rn ✅ docs
* #18325 ✅ rn ✅ docs

## 19 Jul 2023

* #18302 ✅ rn ✅ docs
* #17592 ✅ rn ✅ docs
* #17934 ✅ rn ✅ docs
* #18335 ✅ rn ✅ docs
* #18307 ✅ rn ✅ docs
* #12587 ✅ rn ✅ docs
* #18342 ✅ rn ✅ docs
* #18344 ✅ rn ✅ docs

## 20 Jul 2023

* #18338 ✅ rn ✅ docs
* #18346 ✅ rn ✅ docs
* #18341 ✅ rn ✅ docs
* #18326 ✅ rn ✅ docs
* #18343 ✅ rn ✅ docs
* #18360 ✅ rn ✅ docs
* #17759 ✅ rn ✅ docs

## 21 Jul 2023

* #17502 ✅ rn ✅ docs
* #18353 ✅ rn ✅ docs
* #18177 ✅ rn ✅ docs
* #18004 ✅ rn ❌ docs

## 22 Jul 2023

* #18378 ✅ rn ✅ docs
* #18377 ✅ rn ✅ docs
* #18373 ✅ rn ✅ docs
* #18355 ✅ rn ✅ docs
* #18369 ✅ rn ✅ docs
* #18354 ✅ rn ✅ docs

## 23 Jul 2023

* #18357 ✅ rn ✅ docs
* #18359 ✅ rn ✅ docs
* #18366 ✅ rn ✅ docs
* #16757 ✅ rn ❌ docs
* #17573 ✅ rn ✅ docs
* #18386 ✅ rn ✅ docs
* #16701 ✅ rn ✅ docs
* #18394 ✅ rn ✅ docs
* #18388 ✅ rn ✅ docs

## 24 Jul 2023

* #18403 ✅ rn ✅ docs
* #18400 ✅ rn ✅ docs
* #18385 ✅ rn ✅ docs
* #18409 ✅ rn ✅ docs
* #18407 ✅ rn ✅ docs
* #18419 ✅ rn ✅ docs
* #18415 ✅ rn ✅ docs

## 26 Jul 2023

* #18410 ✅ rn ✅ docs
* #18417 ✅ rn ✅ docs
* #18102 ✅ rn ✅ docs
* #18401 ✅ rn ✅ docs
* #18431 ✅ rn ✅ docs
* #18420 ✅ rn ✅ docs
* #18414 ✅ rn ✅ docs
* #18430 ✅ rn ✅ docs

## 27 Jul 2023

* #18426 ✅ rn ✅ docs
* #18014 ✅ rn ✅ docs
* #17667 ✅ rn ✅ docs
* #18172 ✅ rn ✅ docs
* #18404 ✅ rn ✅ docs
* #18418 ✅ rn ✅ docs
* #18298 ✅ rn ✅ docs
* #18435 ✅ rn ✅ docs
* #18317 ✅ rn ✅ docs
* #18121 ✅ rn ✅ docs

## 28 Jul 2023

* #18444 ✅ rn ✅ docs
* #18445 ✅ rn ✅ docs

## 29 Jul 2023

* #18459 ✅ rn ✅ docs

## 30 Jul 2023

* #18458 ✅ rn ✅ docs
* #18381 ✅ rn ✅ docs

## 31 Jul 2023

* #18372 ✅ rn ✅ docs
* #18432 ✅ rn ✅ docs
* #18469 ✅ rn ✅ docs

## 1 Aug 2023

* #18466 ✅ rn ✅ docs
* #18371 ✅ rn ✅ docs
* #18140 ✅ rn ✅ docs
* #18315 ✅ rn ✅ docs
* #18484 ✅ rn ✅ docs
* #18473 ✅ rn ✅ docs
* #18474 ✅ rn ✅ docs
* #18269 ✅ rn ✅ docs
* #18460 ✅ rn ✅ docs
* #18441 ✅ rn ✅ docs
* #18488 ✅ rn ✅ docs
* #18477 ✅ rn ✅ docs
* #18478 ✅ rn ✅ docs

## 2 Aug 2023

* #18483 ✅ rn ✅ docs
* #18016 ✅ rn ✅ docs
* #18345 ✅ rn ✅ docs
* #18503 ✅ rn ✅ docs
* #18323 ✅ rn ✅ docs
* #18191 ✅ rn ✅ docs
* #18395 ✅ rn ❌ docs
* #18504 ✅ rn ✅ docs

## 3 Aug 2023

* #18500 ✅ rn ✅ docs
* #18505 ✅ rn ✅ docs
* #18512 ✅ rn ✅ docs
* #18454 ✅ rn ✅ docs
* #18524 ✅ rn ✅ docs
* #18480 ✅ rn ✅ docs
* #18528 ✅ rn ✅ docs
* #18532 ✅ rn ✅ docs
* #18374 ✅ rn ✅ docs

## 4 Aug 2023

* #18523 ✅ rn ✅ docs
* #18501 ✅ rn ✅ docs
* #18535 ✅ rn ✅ docs
* #18159 ✅ rn ✅ docs
* #18531 ✅ rn ✅ docs
* #18548 ✅ rn ✅ docs

## 5 Aug 2023

No merges.

## 6 Aug 2023

* #18560 ✅ rn ✅ docs

## 7 Aug 2023

* #18123 ✅ rn ✅ docs
* #18543 ✅ rn ✅ docs
* #18516 ✅ rn ✅ docs
* #18552 ✅ rn ✅ docs
* #18529 ✅ rn ✅ docs
* #18494 ✅ rn ✅ docs
* #18549 ✅ rn ✅ docs
* #18365 ✅ rn ✅ docs
* #18534 ✅ rn ✅ docs
* #18567 ✅ rn ✅ docs
* #18492 ✅ rn ✅ docs
* #18319 ✅ rn ✅ docs

## 8 Aug 2023

* #18427 ✅ rn ✅ docs
* #18577 ✅ rn ✅ docs
* #18517 ✅ rn ✅ docs
* #18382 ✅ rn ✅ docs
* #18569 ✅ rn ✅ docs
* #18576 ✅ rn ✅ docs
* #18479 ✅ rn ✅ docs
* #18575 ✅ rn ✅ docs
* #18530 ✅ rn ✅ docs
* #18527 ✅ rn ✅ docs
* #18541 ✅ rn ✅ docs
* #18571 ✅ rn ✅ docs
* #18583 ✅ rn ✅ docs
* #18200 ✅ rn ✅ docs
* #18587 ✅ rn ✅ docs

## 9 Aug 2023

* #18540 ✅ rn ✅ docs
* #18547 ✅ rn ✅ docs
* #18533 ✅ rn ✅ docs
* #18485 ✅ rn ✅ docs
* #18598 ✅ rn ✅ docs
* #18485 ✅ rn ✅ docs
* #18551 ✅ rn ✅ docs
* #18604 ✅ rn ✅ docs
* #18256 ✅ rn ✅ docs
* #18612 ✅ rn ✅ docs

## 10 Aug 2023

* #18614 ✅ rn ✅ docs
* #18465 ✅ rn ✅ docs
* #18619 ✅ rn ✅ docs
* #18607 ✅ rn ✅ docs
* #18596 ✅ rn ✅ docs
* #18586 ✅ rn ✅ docs
* #18602 ✅ rn ✅ docs
* #18305 ✅ rn ✅ docs
* #18617 ✅ rn ✅ docs
* #18564 ✅ rn ✅ docs
* #18594 ✅ rn ✅ docs

### CODE REVIEW

File: docs/src/main/sphinx/release/release-423.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+# Release 423 (2 Aug 2023)
+
+## General
+
+* Add support for renaming fields with a `RENAME COLUMN` statement. ({issue}`16757`)
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-423.md:5]: Why do we call it field vs column? 

```suggestion
* Add support for renaming columns with a `ALTER TABLE ... RENAME COLUMN` statement. ({issue}`16757`)
```

File: docs/src/main/sphinx/release/release-423.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  a valid URI. Previously, the functions returned ``NULL`` in such cases. ({issue}`18253`)
+* Fix failure to calculate query statistics when a query contains
+  `WHERE ... IN (NULL)`. ({issue}`18328`)
+
+## JDBC driver
+
+* Add support for
+  [constrained delegation](https://web.mit.edu/kerberos/krb5-latest/doc/appdev/gssapi.html#constrained-delegation-s4u)
+  with Kerberos. ({issue}`17853`)
+* Add support for taking action when there are connection parameter validation
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-423.md:23]: What is meant here with "taking action" ?

File: docs/src/main/sphinx/release/release-423.md (hunk 1 above)
Comment (ebyhr) [reply docs/src/main/sphinx/release/release-423.md:5]: Because it renames fields. The syntax is `ALTER TABLE table RENAME COLUMN column.field TO new_field`

Comment (mosabua) [reply docs/src/main/sphinx/release/release-423.md:5]: I might be missing something here.. what is a field of a column? 

In our SQL docs for rename column we have

```
ALTER TABLE [ IF EXISTS ] name RENAME COLUMN [ IF EXISTS ] old_name TO new_name
```

Is the field stuff something else?

File: docs/src/main/sphinx/release/release-423.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
+# Release 423 (3 Aug 2023)
+
+## General
+
+* Add support for renaming fields with a `RENAME COLUMN` statement. ({issue}`16757`)
+* Add support for setting a field type with a `SET DATA TYPE` statement. ({issue}`16959`)
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-423.md:6]: @ebyhr I reworded your suggestion to this

File: docs/src/main/sphinx/release/release-423.md (hunk 2 above)
Comment (mosabua) [reply docs/src/main/sphinx/release/release-423.md:27]: @nineinchnick and @wendigo  .. can you clarify what this means?

File: docs/src/main/sphinx/release/release-423.md (hunk 1 above)
Comment (mosabua) [reply docs/src/main/sphinx/release/release-423.md:5]: oh .. I looked at #16959 now ... this new treatment for nested fields is completely undocumented at this stage so talking about it in the release notes is kinda bad. Anyway ... we need to catch up on this asap. 

@findepi @ebyhr @martint ... I think we need to do a few things:

1. Update ALTER TABLE and others around nested fields, what about CREATE TABLE ?
2. Maybe have some generic doc about nested fields to link to overall
3. Update docs for connectors that support that

Anything else you can think of? Is one of you going to work on this or can at least support me with relevant technical info and review?

Comment (ebyhr) [reply docs/src/main/sphinx/release/release-423.md:5]: In CREATE TABLE statement, we can simply use `column row(field int, ...)`. Currently, only Iceberg connector supports the syntax. 

I can support you. Please feel free to ask me if you have questions.

File: docs/src/main/sphinx/release/release-423.md (hunk 2 above)
Comment (wendigo) [reply docs/src/main/sphinx/release/release-423.md:27]: It just means that exceptions are now more informative and actionable. instead of "invalid argument" you will get exception explaining the problem i.e. "argument a cannot be used when argument c is set"

Comment (wendigo) [reply docs/src/main/sphinx/release/release-423.md:27]: It is user facing but not in a way thar is covered by docs

Comment (nineinchnick) [reply docs/src/main/sphinx/release/release-423.md:27]: Also note that the release notes suggested in the PR were different:
> * Connection parameter validation errors are now actionable ({issue}`12587`)

File: docs/src/main/sphinx/release/release-423.md (hunk 1 above)
Comment (mosabua) [reply docs/src/main/sphinx/release/release-423.md:5]: ok .. I will work on docs .. I also updated the release notes entry .. please have a look once I pushed

File: docs/src/main/sphinx/release/release-423.md (hunk 2 above)
Comment (mosabua) [reply docs/src/main/sphinx/release/release-423.md:27]: I reworded now .. please have a look

File: docs/src/main/sphinx/release/release-423.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
+# Release 423 (9 Aug 2023)
+
+## General
+
+* Add support for renaming nested fields in a column with a `RENAME COLUMN`
+  statement. ({issue}`16757`)
+  * Add support for setting the type of a nested field in a column with a `SET
```
Comment (martint) [inline docs/src/main/sphinx/release/release-423.md:7]: Extra space before `*`

Comment (martint) [reply docs/src/main/sphinx/release/release-423.md:7]: > Add support for setting the type of a nested field in a column via `SET DATA TYPE`.

File: docs/src/main/sphinx/release/release-423.md [hunk 5]
```diff
@@ -0,0 +1,134 @@
+# Release 423 (9 Aug 2023)
+
+## General
+
+* Add support for renaming nested fields in a column with a `RENAME COLUMN`
```
Comment (martint) [inline docs/src/main/sphinx/release/release-423.md:5]: > Add support for renaming nested fields in a column via `RENAME COLUMN`

File: docs/src/main/sphinx/release/release-423.md [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+  statement. ({issue}`16757`)
+  * Add support for setting the type of a nested field in a column with a `SET
+  DATA TYPE` statement. ({issue}`16959`)
+* Add support for column comments on materialized views. ({issue}`18016`)
```
Comment (martint) [inline docs/src/main/sphinx/release/release-423.md:9]: > Add support for comments on materialized view columns.

File: docs/src/main/sphinx/release/release-423.md [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+* Improve performance of `INSERT` and `CREATE TABLE AS ... SELECT` queries. ({issue}`18212`)
+* Improve performance when planning queries involving multiple window functions. ({issue}`18491`)
+* Improve optimizer estimates for queries with a `BETWEEN` clause. ({issue}`18501`)
```
Comment (martint) [inline docs/src/main/sphinx/release/release-423.md:12]: What's the user-visible effect? Maybe rephrase to:

> Improve performance of queries involving `BETWEEN` clauses.

File: docs/src/main/sphinx/release/release-423.md [hunk 8]
```diff
@@ -0,0 +1,134 @@
...
+* Improve performance of queries containing redundant `ORDER BY` clauses in
+  views or `WITH` clauses. This may affect the semantics of queries that
+  incorrectly rely on implementation-specific behavior. The old behavior can be
+  restored via the `skip_redundant_sort` session property or the
+  `optimizer.skip-redundant-sort` configuration property. ({issue}`18159`)
+* Reduce default values for the `task.partitioned-writer-count` and
+  `task.scale-writers.max-writer-count` configuration properties to reduce the
+  memory requirements of queries that write data. ({issue}`18488`)
+* Reduce DNS lookup overhead on NodeScheduler. ({issue}`18365`)
```
Comment (martint) [inline docs/src/main/sphinx/release/release-423.md:21]: What's the user-visible effect? NodeScheduler is an internal implementation detail.

File: docs/src/main/sphinx/release/release-423.md [hunk 9]
```diff
@@ -0,0 +1,134 @@
...
+* Disallow coercing `timestamp` types to `varchar` for historical dates. ({issue}`18004`)
```
Comment (martint) [inline docs/src/main/sphinx/release/release-423.md:22]: * What's a "historical date"? 
* There are no implicit casts between timestamp and varchar in the engine -- this change is specific to Hive, but in what context does it apply? E.g., for partition columns? When column metadata doesn't match data in files, etc?

File: docs/src/main/sphinx/release/release-423.md [hunk 10]
```diff
@@ -0,0 +1,134 @@
...
+* Remove the deprecated `optimizer.use-mark-distinct` configuration property,
+  which has been replaced with `optimizer.mark-distinct-strategy`. ({issue}`18540`)
+* Fix query planning failure due to dynamic filters in
+  [fault tolerant execution mode](/admin/fault-tolerant-execution). ({issue}`18383`)
+* Fix URI extraction functions (`uri_extract_...`) ignoring input values that
```
Comment (martint) [inline docs/src/main/sphinx/release/release-423.md:27]: This is a backward incompatible language change that needs more consideration before we can get it in. We're going to need to revert this for the release.

File: docs/src/main/sphinx/release/release-423.md [hunk 11]
```diff
@@ -0,0 +1,134 @@
...
+  are not a valid URI. Previously, the functions returned ``NULL`` in such
+  cases. ({issue}`18253`)
+* Fix failure to calculate query statistics when a query contains
```
Comment (martint) [inline docs/src/main/sphinx/release/release-423.md:30]: What is the visible impact of "failing to calculate statistics"? Does the query fail? Are the stats in presented to the connector incorrect?

File: docs/src/main/sphinx/release/release-423.md [hunk 12]
```diff
@@ -0,0 +1,134 @@
...
+  `WHERE ... IN (NULL)`. ({issue}`18328`)
+
+## JDBC driver
+
+* Add support for
+  [constrained delegation](https://web.mit.edu/kerberos/krb5-latest/doc/appdev/gssapi.html#constrained-delegation-s4u)
+  with Kerberos. ({issue}`17853`)
+* Improve error messages from connection parameter validation. ({issue}`12587`)
```
Comment (martint) [inline docs/src/main/sphinx/release/release-423.md:38]: We generally don't document improvements to error messages in the release notes.

File: docs/src/main/sphinx/release/release-423.md [hunk 13]
```diff
@@ -0,0 +1,134 @@
...
+  table properties. ({issue}`17294`)
+* Add support for reading the `timestamp_ntz` type. ({issue}`17502`)
+* Add support for writing the `timestamp with time zone` type on partitioned
+  columns. ({issue}`16822`)
+* Add option to enforce that a filter on a partition key is present for
+  query processing. This can be enabled by setting the
+  ``delta.query-partition-filter-required`` configuration property or the
+  ``query_partition_filter_required`` session property to ``true``.
+  ({issue}`18345`)
+* Improve performance of the $history system table. ({issue}`18427`)
```
Comment (martint) [inline docs/src/main/sphinx/release/release-423.md:62]: `$history` (missing backquotes)

File: docs/src/main/sphinx/release/release-423.md [hunk 14]
```diff
@@ -0,0 +1,134 @@
...
+  [identity columns](https://github.com/delta-io/delta/blob/master/PROTOCOL.md#identity-columns). ({issue}`18200`)
+* Fix incorrectly creating files smaller than the configured
+  `file_size_threshold` as part of `OPTIMIZE`. ({issue}`18388`)
+* Fix query failure when a table has a file with a location ending with
+  whitespace. ({issue}`18206`)
+
+
+## Hive connector
+
+* Add support for `varchar` to `timestamp` coercion. ({issue}`18014`)
```
Comment (martint) [inline docs/src/main/sphinx/release/release-423.md:73]: In what context? See comment above

File: docs/src/main/sphinx/release/release-423.md [hunk 15]
```diff
@@ -0,0 +1,134 @@
...
+* Remove the legacy Parquet writer, along with the
+  `parquet.optimized-writer.enabled` configuration property and the
+  `parquet_optimized_writer_enabled ` session property. Replace the
+  `parquet.optimized-writer.validation-percentage` configuration property with
+  `parquet.writer.validation-percentage`. ({issue}`18420`)
+* Fix incorrectly creating files smaller than the configured
+  `file_size_threshold` as part of `OPTIMIZE`. ({issue}`18388`)
+* Fix loss of data precision when coercing `timestamp` values. ({issue}`18003`)
```
Comment (martint) [inline docs/src/main/sphinx/release/release-423.md:81]: Coercing to what? In what context?

File: docs/src/main/sphinx/release/release-423.md [hunk 16]
```diff
@@ -0,0 +1,134 @@
...
+## Hudi connector
+
+* Fix incorrectly creating files smaller than the configured
+  `file_size_threshold` as part of `OPTIMIZE`. ({issue}`18388`)
+* Fix query failure when a table has a file with a location ending with
+  whitespace. ({issue}`18206`)
+
+## Iceberg connector
+
+* Add support for renaming nested fields in a column with a `RENAME COLUMN`
```
Comment (martint) [inline docs/src/main/sphinx/release/release-423.md:98]: > ... via `RENAME COLUMN`

**Review by martint — CHANGES_REQUESTED** [review]

File: docs/src/main/sphinx/release/release-423.md (hunk 4 above)
Comment (mosabua) [reply docs/src/main/sphinx/release/release-423.md:7]: Done

File: docs/src/main/sphinx/release/release-423.md (hunk 5 above)
Comment (mosabua) [reply docs/src/main/sphinx/release/release-423.md:5]: Done

File: docs/src/main/sphinx/release/release-423.md (hunk 6 above)
Comment (mosabua) [reply docs/src/main/sphinx/release/release-423.md:9]: Done

File: docs/src/main/sphinx/release/release-423.md (hunk 7 above)
Comment (mosabua) [reply docs/src/main/sphinx/release/release-423.md:12]: yeah .. that sounds good. Done.

File: docs/src/main/sphinx/release/release-423.md (hunk 8 above)
Comment (mosabua) [reply docs/src/main/sphinx/release/release-423.md:21]: From the PR it is maybe a small performance gain .. but not sure if we should call it that..

File: docs/src/main/sphinx/release/release-423.md (hunk 9 above)
Comment (mosabua) [reply docs/src/main/sphinx/release/release-423.md:22]: Have to look at the PR some more to see if I can get that info.

File: docs/src/main/sphinx/release/release-423.md (hunk 10 above)
Comment (mosabua) [reply docs/src/main/sphinx/release/release-423.md:27]: Removed

File: docs/src/main/sphinx/release/release-423.md (hunk 11 above)
Comment (mosabua) [reply docs/src/main/sphinx/release/release-423.md:30]: Have to look some more at the PR.

File: docs/src/main/sphinx/release/release-423.md (hunk 12 above)
Comment (mosabua) [reply docs/src/main/sphinx/release/release-423.md:38]: In this case its pretty significant to from just a failure to actual error message that say what properties and values are wrong. This is part of the JDBC URL support for the CLI and that PR is already mentioned.

File: docs/src/main/sphinx/release/release-423.md (hunk 13 above)
Comment (mosabua) [reply docs/src/main/sphinx/release/release-423.md:62]: Done

File: docs/src/main/sphinx/release/release-423.md (hunk 8 above)
Comment (mosabua) [reply docs/src/main/sphinx/release/release-423.md:19]: Seems like this is a performance and stability gain from the PR .. but I might be reading to much into the conversation in the PR @martint

File: docs/src/main/sphinx/release/release-423.md (hunk 9 above)
Comment (mosabua) [reply docs/src/main/sphinx/release/release-423.md:20]: Historical dates are before 1900 in this PR. Not sure about what context .. seems to be generic, at least from just looking at the PR.

File: docs/src/main/sphinx/release/release-423.md (hunk 11 above)
Comment (mosabua) [reply docs/src/main/sphinx/release/release-423.md:25]: This seems to be two fixes. In both cases running EXPLAIN on a query fails .. but not sure if the actual query would fail or what the effect is.

File: docs/src/main/sphinx/release/release-423.md (hunk 9 above)
Comment (martint) [reply docs/src/main/sphinx/release/release-423.md]: Then we should call it out as "dates before xxx".

File: docs/src/main/sphinx/release/release-423.md (hunk 12 above)
Comment (colebow) [reply docs/src/main/sphinx/release/release-423.md:33]: The theory is that these changes are self-documenting, and users will gain the benefits of the change without needing to know anything or take any action.

File: docs/src/main/sphinx/release/release-423.md [hunk 17]
```diff
@@ -5,6 +5,7 @@
 * Add support for renaming nested fields in a column via `RENAME COLUMN`. ({issue}`16757`)
 * Add support for setting the type of a nested field in a column via `SET DATA TYPE`. ({issue}`16959`)
 * Add support for comments on materialized view columns. ({issue}`18016`)
+* Show original characters for string literals. ({issue}`5061`)
```
Comment (colebow) [inline docs/src/main/sphinx/release/release-423.md:8]: I'm not really sure what this means.

Comment (mosabua) [reply docs/src/main/sphinx/release/release-423.md:8]: Yeah .. not sure how to reword and also where it shows up .. the PR talk about displaying unicode symbols used for literals when doing a SHOW CREATE TABLE .. but I have no idea how wide the scope is .. for example I think it will also apply for SHOW CREATE SCHEMA or SHOW CREATE VIEW and so on..

File: docs/src/main/sphinx/release/release-423.md (hunk 11 above)
Comment (martint) [reply docs/src/main/sphinx/release/release-423.md:24]: In that case, we should rephrase to "Fix EXPLAIN failure when ..."

Comment (mosabua) [reply docs/src/main/sphinx/release/release-423.md:24]: Done

File: docs/src/main/sphinx/release/release-423.md (hunk 9 above)
Comment (mosabua) [reply docs/src/main/sphinx/release/release-423.md]: Done
